pages/common/promtool.md

```diff
@@ -0,0 +1,20 @@
+# promtool
+
+> Tooling for the Prometheus monitoring system.
+> More information: <https://prometheus.io/docs/prometheus/latest/getting_started/>.
+
+- Check if the config files are valid or not (if present report errors):
+
+`promtool check config {{config_file}}.yml`
+
+- Check if the rule files are valid or not (if present report errors):
+
+`promtool check rules {{rules_file}}.yml`
+
+- Pass Prometheus metrics over stdin to check them for consistency and correctness:
+
+`curl -s {{http://example.com:9090/metrics/}} | promtool check metrics`
```

Comment:
It's best to use the long options for commands when possible, to be more descriptive, even though this page isn't actually about `curl`.
```suggestion
`curl --silent {{http://example.com:9090/metrics/}} | promtool check metrics`
```